src/System.Private.Uri/tests/FunctionalTests/IdnDnsSafeHostTest.cs

```diff
@@ -45,6 +45,21 @@ public void IdnDnsSafeHost_IPv6Host_ScopeIdButNoBrackets()
             Assert.Equal("[::1]", test.Host);
         }
 
+        [Theory]
```

Comment:
We usually use `[Theory]` for multiple test data. So here 
1. you can add more test data (`[InlineData]`)
2. or use `[Fact]`, without taking any parameters, and create a constant in the test. As David [mentioned here](https://github.com/dotnet/corefx/pull/29829#discussion_r189685517).

Reply:
fixed